src/BibleWell.App/Views/Pages/LanguagesPageView.axaml

```diff
@@ -35,6 +35,29 @@
                 </DataTemplate>
             </ListBox.ItemTemplate>
         </ListBox>
+
+        <StackPanel Margin="20">
```

Comment:
This is just a PoC of fetching all languages from the API and displaying the data.  We might need the languages route for the app but I'm guessing we'll just have languages baked in to the localization files and thus a static list of ones we support.